src/config.rs

```diff
@@ -221,13 +221,19 @@ impl BuilderConfig {
     pub async fn connect_additional_broadcast(
         &self,
     ) -> Result<Vec<RootProvider<BoxTransport>>, ConfigError> {
-        let mut providers = Vec::with_capacity(self.tx_broadcast_urls.len());
-        for url in self.tx_broadcast_urls.iter() {
-            let provider =
-                ProviderBuilder::new().on_builtin(url).await.map_err(Into::<ConfigError>::into)?;
-            providers.push(provider);
+        if !self.tx_broadcast_urls.is_empty() && self.tx_broadcast_urls[0] != "" {
```

Comment:
should we just do this when we actually populate `self.tx_broadcast_urls`?

Reply:
like this? https://github.com/init4tech/builder/pull/50/files

Reply:
Sure, that works too. I was thinking about the case where we wanted to do additional setup or validation with the URLs but this works.